src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -1548,6 +1547,7 @@ namespace AirflowNetworkBalanceManager {
         using DataLoopNode::NodeConnectionType_Inlet;
         using DataLoopNode::NodeType_Air;
         using DataLoopNode::ObjectIsParent;
+        using DataSurfaces::SurfWinOriginalClass;
```

Comment:
Adding more `usings` isn't great.  Probably not worth more commits, but we are moving away from it.  Even if it makes the lines of code a little wider...that's hardly a problem on most folks monitors these days.